packages/api-queries/src/me-notifications-settings.ts

```diff
@@ -0,0 +1,28 @@
+import { fetchMeNotificationSettings, updateWpcomNotificationSettings } from '@automattic/api-core';
+import { queryOptions, mutationOptions } from '@tanstack/react-query';
+import { queryClient } from './query-client';
+
+export const meNotificationsSettingsQuery = () =>
```

Comment:
[nit] for consistency with other `me-*` queries, let's use `userNotificationSettingsQuery()` rather than `meNotificationSettingsQuery()` (also the mutation)